doc/rados/operations/health-checks.rst

```diff
@@ -1316,35 +1319,41 @@ The location of an OSD can be found with:
 PG_NOT_SCRUBBED
 _______________
 
-One or more PGs has not been scrubbed recently.  PGs are normally scrubbed
-within every configured interval specified by
-:confval:`osd_scrub_max_interval` globally. This
-interval can be overridden on per-pool basis with
-:confval:`scrub_max_interval`. The warning triggers when
-``mon_warn_pg_not_scrubbed_ratio`` percentage of interval has elapsed without a
-scrub since it was due.
+One or more Placement Groups (PGs) have not been scrubbed recently. PGs are
+normally scrubbed within every configured interval determined by
```

Comment:
Suggest `within an interval determined by`

As written this connotes to me that multiple specific intervals are configured.  Yes this is subtle.

Reply:
It's a good point.